test/e2e/run_test.go

```diff
@@ -1205,21 +1205,21 @@ USER mail`, BB)
 	})
 
 	It("podman run with bad healthcheck retries", func() {
-		session := podmanTest.Podman([]string{"run", "-dt", "--health-cmd", "[\"foo\"]", "--health-retries", "0", ALPINE, "top"})
+		session := podmanTest.Podman([]string{"run", "-dt", "--health-cmd", "foo", "--health-retries", "0", ALPINE, "top"})
```

Comment:
```suggestion
		session := podmanTest.Podman([]string{"run", "-dt", "--health-cmd", "foo", "--health-retries", "0", ALPINE, "top"})
```
With this change does the health-retries error out as expected or the health-cmd not being an array?

Reply:
healthCmd is read in as a string and then converted to an array so as long as you pass it a single string with no quotes: `foo` or a long string with quotes `"foo bar 123"` the errors don't change

Reply:
This is a breaking change. Why should `[\"foo\"]` no longer be valid?

Reply:
I can change the parsing so [] is valid. Personally it doesn't make sense to me why we would take an array if our healthchecks on the cmd level are strings, [""] and "" are just as easy to input

Reply:
The point is this is breaking users who already use the `[""]` syntax. Breaking changes should be limited to major releases. It is also documented in the man page that this syntax is valid.

Reply:
just changed it so `[""]` is valid @Luap99

Reply:
I think you have to use `json.Unmarshal` as this will handle special escape sequences and more. Also I do not think just removing the first bracket will not work for `["some","long","command"]`

Reply:
It removes both brackets. The array is technically a string so long commands wont matter (I don't think) as it reads it as one consecutive string.

Reply:
`strings.Fields(inCmd)` will only split at whitespaces but in this case it has to split at the comma and also remove the quotes. Therefore you should use `json.Unmarshal`.

Reply:
resolving now.